### ISSUE TITLE
"OpenBVE" naming discussion (relative #325 and #305)

### ISSUE THREAD

**Issue by ginga81** [leezer3/OpenBVE#341]
At
https://github.com/leezer3/OpenBVE/commit/9e626c63be33d7ca45fcea5ba816ffbe9f221e6b#diff-87f0cc57cdf329fea9aac9ffed1e84d2

Why are you changed the name 'OpenBVE' to 'openBVE'?
Especially in Japan, the naming of 'BVE' is extremely sensitive.
So, the naming is very important.

'BVE' is the category of Train Simulater for a long time we have called.
At the history, BVE2/4 calls 'BVE'.
From Bve5, the name changed to 'Bve'.
Now, we are attempt to loading the Bve5's data, but this is different approach.

OpenBVE's data is compatible with BVE2/4.
Additionally, OpenBVE is open source software.
(Now we are attempt to change BSD)
The name of openBVE is more emphasis of 'BVE'.
Surely, OpenBVE can run with BVE2/4's data and train.
But, openBVE is not based from BVE2/4.
'OpenBVE' is an one name.
Please do not sepalate from 'BVE'.

So, I suggest 'OpenBVE' at #325 .

Please description why are you decision the name of 'openBVE'.
And very deeply discuss about the naming of 'OpenBVE'.

**Comment by leezer3** [comment]
The 'correct' spelling is (probably at least) openBVE , although both have been used interchangeably in the past.

Original site from archive.org (note spelling): http://web.archive.org/web/20090415042415/http://openbve.trainsimcentral.co.uk/

This stems from the fact that in English, it's technically gramatically incorrect for a proper name / start of a sentence to start with a lower case letter, hence openBVE often gets auto-corrected to OpenBVE.

OpenBve is used in the source code due to C# CamelCase naming conventions. At this point it's rather hard to change that :)

**Comment by leezer3** [comment]
As an addendum:
I strongly suspect we're stuck with openBVE.

Changing a name is not an easy process in the first place, and there's 15 years behind this one. Complicating matters considerably is the diverse nature of the community and the sheer number of sites providing content for the sim.
It'd be impossible to get half of the active ones to refer to the 'new' name, let alone the inactive ones.....

**Comment by ginga81** [comment]
We want to use the name of 'OpenBVE'.
Surely, as you say, the original is openBVE.
But, at Japanese Bve5's community is extremely sensitive the name of 'BVE'.

Some people says that "Do not use BVE's name. openBVE is not BVE".

Also as you say, the English grammar, at the start of a sentence to start with a lower case letter is not better.
To fit the English grammar, and evacuate the trouble of naming with comunity, please change to 'OpenBVE'.

I think that the sourcecode' s OpenBve is not to problem because to obey the C# naming.

The original Bve5's history is 20 years.
And this year is Bve's 20th memorial.

Even though a half of people cannot read the new name, we want to change.
That's a very important issue.

**Comment by s520** [comment]
I have a strong interest in this argument.
I think that the designation in the document etc should be unified to "Open BVE". (not "openBVE")
The impact of this change on search engines is extremely limited :)

**Comment by leezer3** [comment]
I think the change would be more than you anticipate actually.

Changing from openBVE to OpenBVE (retaining the one-word structure) would have no appreciable affect, and I think would only add to the confusion.
 
I would agree that the immediately visible affect on search engines of moving to the two-word format Open BVE is likely to be limited to none, but the trouble is somewhat more that introducing a second word means far more potential for mis-typing, mis-spelling and so-forth.

For that matter, all three variants (openBVE , OpenBVE and Open BVE) are already used somewhat interchangeably in the wider community, and as far as I can see, the problems you're facing are all related to the use of the three letters **BVE** ; From 20 years of experience of this sort of argument, I can almost guarantee that the change you're advocating will get ignored totally :)

**Comment by leezer3** [comment]
Other thoughts-
If you want to refer to any of the three variants in the Japanese translations etc. you're welcome to; As noted above, they're interchangable.

It's also apparent that this is a Japanese problem; Outside Japan, the situation is almost reversed, whereby we are the 'real' thing...

For what it's worth, these were yesterday's visitor stats for the main site:


| Country | Visitors |
|---|---|
| United States | 1,111 |
| China | 637 |
| United Kingdom | 386 |
| Japan | 344 |
| Hungary | 326 |

**Comment by s520** [comment]
Oh, I'm sorry.
What I intended was a change to "**OpenBVE**".
Extra space was inserted by Google translation.
I know that various notations are used in the community.
I think that we should unify the **official** description method.

**Comment by ginga81** [comment]
'Outside of Japan, the situation is almost reversed'

Sure, it is a natural result.

Do you know who says that hope do not use 'BVE' name for openBVE?
'mackoy', the original BVE creator.
https://mackoy.exblog.jp/i4/

Also at here, mackoy says that he hope do not use logo-mark for openBVE.

So,  most of Japanese BVE5's comunity people, especially 15 years or longer data creaters are extremely hate to openBVE.

At the our new works of OpenBVE, I hope that their mind are a bit changed.
This is our about four years contacting and continuing to comunicate very cafully.

As you know, 'the ORIGINAL THREE' are battled with original BVE community.
And at the 1.4.3, they are retired.
Recently, we are attacked from the older community's member again.
Because of these, all of related.

The Japanese BVE's community is operateing by mainly boss of the oldest creators.
They are continueing to create data and leading community.

Certainly, mackoy's wish (All name (with BVE) changes) is incredibly difficult, but a set of the project name of 'OpenBVE' is different 'Bve'.
OpenBVE can read the BVE2/4's data, and BVE2/4 are calls 'BVE', and now, Bve5 is not call mainly 'BVE'.
I think that the 'BVE' name is a categolize of Trainsim.
At least, set of project name of 'OpenBVE', naming head letter is 'O', so more weakly impact of English grammar, is it?
openBVE is more strongly emphasize with the 'BVE'.

From the above,  a long history of Japanese BVE are very deeply problem.

But now, you are talking only using the name for a long time, and only to talking with access counter.
Perhaps, you think that bothersome with change name? 

Perhaps you are not know these truth history and the truth of Japanese original BVE's comunity's.
Which is more important do you think?

Changing of the licensing is the best timing of change the logo-mark and naming.
So, I suggest at #325 (logoamark change with change to OpenBVE) and #305(licensing).

So, For the future of the project, I think strongly that change the name to OpenBVE and change the logo-mark.

- If you do not want battle with mackoy and BVE5's community, and you do not want distarb our create -

**Comment by ginga81** [comment]
I know someone who pushed bad button.
But now, we are discussing, isn't it ?
If someone who are thinking of something, if only to push bad icon, I cannot understand what is bad.
Please write.
I am not saying I want to do not use 'openbve' name, can you understand?

**Comment by leezer3** [comment]
I'm aware of the basics of the situation, although not so much in detail.

Unfortunately as noted above, my blunt assessment is that a casing change in the existing name will do absolutely nothing, and would instead just serve to confuse things further.
I would also strongly suspect that the hate would follow any changed name; Things like that are irrational that way.

So-
* At this point, we've got ~15 years of history with the current name.
* There has already been one (unsuccessful) attempt to change the name by Michelle.
* I'm by no means sure changing the name will solve your problems.
* Confusion between BVE and openBVE is relatively low.

Basically, I feel that to even consider a name change and attempt to solve your issues, we would need to make a clean break with the current.& drop the BVE moniker.
Good names are *hard* to come by....

**Comment by ginga81** [comment]
- At this point, we've got ~15 years of history with the current name.

SO, I always to say the timing is change to BSD.
You are obey the ORIGINAL THREE, if so, we must not change the license too, isn't it?
The Bve5's community is still notl accept and forgive, they are only give up to conversation with openBVE's community because openBVE's comunity are not understand.
If something happen again, they will angly again.
Their request is mainly two.
1.name change ->the ORIGINAL THREE choosen 'openBVE'. 
 After discussed, difficult to remove BVE, but changes to 'OpenBVE'.
2.logo-mark ->at the timing of change license to BSD: compedition and change.

- There has already been one (unsuccessful) attempt to change the name by Michelle.

I am not say about completely another naming.
I want to change to 'OpenBVE'. only 'o' to 'O'.

- I'm by no means sure changing the name will solve your problems.

I already know that. But the Japanese Bve5's 
community is understand difficult to discuss to change completely.
So, we have to send message of  "we have been experience the long history. Unfortunately, the 'BVE' cannot remove, but we are discuss and changed from the first name of 'openBVE'".

- Confusion between BVE and openBVE is relatively low.

This topic is not to say about naming confusion.
Project's naming.

After 15 years, at 1.6.0, if we will return to debian, OpenBVE is changed evolutionaly.

And I am saying that I want to change to 'OpenBVE' at the timing of change to BSD.

You say that you are understand this history.
If nothing to change, at least logo-mark, in the future, they will atack to us again, and will difficult to commit the Bve5's route parser.
I am strongly concern about that they perhaps say to us that "Must not loading Bve5's route! Because openBVE is not BVE!".

If this situation happens, what you do to us?
I hope to this concern will not happen.

**Comment by leezer3** [comment]
The trouble is that nothing is simple and easy, and everything we do has a ramification on both us and the wider community.

As noted in the other issue, I'd personally like to change to BSD, and in my view, we could probably legally do this.
On the other hand, doing so is potentially problematic without the specific written consent of the remaining member of the original three (Michelle).

Similiarly, changing the name may bring potential benefits from the Japanese community (although in my opinion most of the problems will likely follow the new name), but has to be balanced against what the current name gives us.

Is there a right and a wrong answer to either of these?

I don't really know.....

At the minute though, I really feel that you'll find out changing the casing of the current name will do absolutely nothing for your issues, but instead will just serve to add fuel to the fire of those who disagree with you.
Furthermore, it's a change which is so minor I really feel I can't support it personally....

I also suspect that your concerns over the BVE5 parser are absolutely justified, and will probably give your critics ammunition no matter what name or logo we go by.

Again, I don't have a good answer to what to do about this (other than ignoring it).

**Comment by ginga81** [comment]
As you say, our action is the very sencitive effect to the both community.
And, it is very difficult to change the project name.
So, I am waiting for the timing of change to the license to BSD.

Michelle's try of changing name is completely new and another name isn't it?
As this case, basic name is keeping, but a part of the name changed?

If Michelle changes the part of name only, but the community says no, I do not say about the name.
But, I think that Michelle tryed to the completely new name.

In the Japanese culture, at the our case of BVE community's trouble, ignoring is the worstest comunication.

Basically, such as the war, to agree with the (enemy)'s request is the worst action.
At the Japanese culture, these case is the same(such as war or law trouble etc.) .

But, when the normal culture comunication, such as BVE, if the community take someting of trouble,  to show the action of something is very good effect to the community.
These action's effect name calls '誠意を見せる'.

If takes the action of with utmost effort, the another community looks all of actions.
And thinking deeply, recognize, and finally, thinking the opposite side's opinion.

And finally, if the community agreement with the part of oppsite community, they say that "your community is very deeply thought about this problem, so our community is forgive it partly,
if your community promice to our community that do not takes the same trouble again".

As you know, the community is not forgive it all, but they know all of process totally why the opposite side of community takes such action finally.

So, at the future, If the comunity's someone says 'this is a problem', the oppositeside of  comunity to stops to attack.
Because, they know the problem's all of process.

Perhaps this culture is very local of Japanese, so perhaps you cannot agree with.
So, I think that you are write above.

If our community takes an action such as logo-mark changes, the Bve5's community knows all of process.
So they are not agreement with, but they are thinking of all, so they will forgives our actions.
These process is important for the future.

In our case, the openBVE comunity is continuing ignoring.
But after 15 years, at the timing of license to BSD, openBVE changes  evolutionally and name is changes to OpenBVE, and changes the logo-mark by compedition.
If we takes these actions, they are looking all of process with 15 years.
They will not to say us(to 1.5' s new committers).

**Comment by leezer3** [comment]
Your cultural points are interesting; I'm intellectually aware of them, but have never really had to apply them in practice.

In Western culture, things are somewhat inverted, whereby any change on our part would / could be seen as weakness, although to be honest it's not really something that would escalate nearly as far as you're implying. If it did, it would I suppose end up with both sides agreeing some degree of a compromise after one had 'won'.......


Michelle:
She attempted to change to Trainsimframework, and this was badly rejected by the community. Some of the pushback from this was partly the reason why she left.


I haven't really got an answer to the issues, other than raising a different set of cultural issues in turn......
Whilst the logo is much more easily changable than the name, designing something that looks good is a much more difficult thing..... 

Perhaps as a first step work towards a logo redesign?

**Comment by ginga81** [comment]
My prediction is matched.
Ms.Michelle, the one of the ORIGINAL THREE was attempt to change the completely difference name of Trainsimframework...
I think perhaps the Japanese Bve5' s community was not know her action.
Because, they were 'battling', and very heating up.
At this timing, they will aware this history.

Recently, I awared that Ms.Michelle is a women, as you writing 'her'.
Her heart is may be broken...

The Western culture is also  I'm intellectually aware of them, but also I have never faceing before.
When invited from you about four? years ago, there is the first experience for me that the difference culture with Western and Japanese.

The Western style of 'battle', in Japan, at the retired of 1.4.3, the battle is end, and Bve5 won completly.
As you said, the homepage access is very low from Japan.
It is the 'battle' is already ended.
After the end of 'battle' of 1.4.3, Japanese community is retired completely.
After then, I invited by you.
As I said, they know that I am not the 1.4.3's community member.

So, they are very soften to the new 'OpenBVE's'  community members.

Recently, the Bve5's very important, distinctive topic of 'the another train' can do by TrackFollowingObject. 
When we implementationed TFO, in Japanese Twitter, the older creators are attacked us.
But this attack is more soften the past years.
As I said before, our actions are very carefully.
So they are already know, and the attacking is more soften from the past 'battle'.
This histoly is very sensitive at Japan, and not obey the guideline, so it will not written to Wikipedia forever.

This time I wrote these hidden history at Japan, this history is very important why I talking about naming.

As we know, the Bve Trainsim is created by mackoy from 20 years ago, from Japan.
This year, 2019 is the 20th memorial year of Bve Trainsim.
As these, at the returning to the Debian is bad timing this year, we have to return to Debian rapidly.

The parser of the Bve5 is very high performance.
I think that it will suprise you.
Additionaly, at the Bve5 parser, there is the new programmar comes in.
We have to protect them.

If our side takes the action, as I said, they looks our all actions and recognize of the difference between with Western and Japanese at these full open discussion.
The Bve5's Japanese community can aware of these actions.

If we takes the actions of naming change to a set of project name of 'OpenBVE' and change the logo-mark at changing of the license to BSD, I can not say anything definite, but it will effect to more soften as I said.

So, at this timing, changing the naming is very important.
'o' to 'O' is very important to show our action.
I wish to change to the name and logo-mark as these.
At least, please get back this change quickly.
https://github.com/leezer3/OpenBVE/commit/9e626c63be33d7ca45fcea5ba816ffbe9f221e6b#diff-87f0cc57cdf329fea9aac9ffed1e84d2

**Comment by leezer3** [comment]
I think you'll find things are far more complicated than you imagine, hence my extreme reluctance to start messing with names and licensing without **extremely** good reasons and justifications for doing so.
The biggest thing I think you're missing, is that changes which 'please' the Japanese community have a danger of doing the opposite elsewhere.

Myself, I was never exactly invited or anything, it was more a case of being the only person who was bothered / technically able enough to start working on the main program, and as such have ended up as the defacto person in charge, and as such I need to think of the bigger picture. This is something Michelle failed to do, and in some ways left us in this situation.....

The whole situation with Michelle is a very complicated one- She fell out with a lot of the community over her attitude towards copyright (hence our current license mess), and she was very resistant towards fixing compatibility issues and making changes to make existing stuff 'work'
The change to Trainsimframework, and the direction she attempted to take before she left was to move away from BVE entirely, and to rather load everything....


Japan / BVE5:
I've obviously had dealings with both you and S520, and some private discussions with others.
350 visitors a day from Japan isn't at all bad, and certainly isn't low, especially for a site written primarily in English.
In many ways, it very much strikes me that our problem is not with the end-users, but rather with the content creators. 
With this in mind, I suspect that things such as translating the developer documentation are likely to have a much better affect than playing with the casing of the name.
It might also be worthwhile getting Henrich to comment on how he sees the name / issues as a Japanese outsider?

**Comment by s520** [comment]
As you say, I think it's good to ask a third party for an opinion.
But Mr. Henrich will not know this problem at all.
It would be better to have some knowledge of BVE5 and OpenBVE.

**Comment by rekuto02** [comment]
mackoyさんは　BVE Trainsim とは別物だと思いました。
と発言しています。
openBVEだと日本人などの中ではBVEと思う方も多いと思います。
ただ、openBVEだと　ginga81　さんが言ったようにどうしてもBVEが強調されてしまいます。
openBVEの名前を捨てて別の名前に変えろ　とは言いません。
oからOに変えてくれればいいのです。(これも　ginga81　さんと同じ)
それも大変なことは私が理解しています。

**Comment by ginga81** [comment]
Based on the previous discussions, I(ginga81), S520, F81_tec200 and Japanese OpenBVE's community had a discussion about these.
At the past time, when we released the TFO, we faced the serious trouble between with Japanese Bve5's community.
So, we decided that we are pending the Bve5 parser's PR to avoid with the community's trouble while the naming issue and logo-mark's problems was not resolve.
And we decided to question to BVE Worldwide forum what these discussion and decides.
If at the Forum opinion can not get consent, at the worstest case, we will decide to stop to release the Bve5 parser.
For your reference, there is the sample running test with 'another train' by Bve5 parser.
Please check it.
https://youtu.be/ubApYNPEZQI
The Bve5 route is the default keisei line created by mackoy.
The train is for BVE4's keisei train also created by mackoy.

**Comment by leezer3** [comment]
Very nice.

You're free to do what you wish with the BVE5 parser, and I'll support you in that. If I can do anything to help, let me know.
That was another of the issues with Michelle's management of the original project, in that there was never enough discussion of the way to go, it was either her way or not at all.

BVE Worldwide topic:
http://bveworldwide.forumotion.com/t1705-name-logo-and-other-complicated-ideas#18074

**Comment by s520** [comment]
Thank you for your kind words.
I hope a solution that both can be convinced.

Let's look at your link from now on.

**Comment by s520** [comment]
Now, discussions at the forum have stopped for two months.
The discussion seems to be exhausted, so let's decide the direction.

As far as the discussion is concerned, many people agree with the handling of "O".
In addition, logo changes are accepted.

Let me know your current opinion again.

I am looking forward to a good reply from you.

**Comment by leezer3** [comment]
Main site changed hopefully.

I can't say I'm convinced by it, but it's not my decision alone.

Need to sort out where it's used in the source code, but that may take a little more time to check as I don't really want to do a global replace on that....

**Comment by s520** [comment]
Thank you for your great decision.

I will submit a PR for the BVE5 scenario parser if the logo is expected to change.

What about the logo?

**Comment by leezer3** [comment]
Happy to look at anything you've got logo wise; 
I think the best way to deal with it will end up being to pass back to the community for a decision on any candidates / changes.

**Comment by s520** [comment]
I propose the following selection procedure.
1. Ask the forum to submit a draft logo.
2. We narrow down to some extent. (To maintain quality)
3. Vote on the voting site. (ex: https://www.surveymonkey.com/)
4. Final decision

(2. may be unnecessary.)

**Comment by leezer3** [comment]
That sounds fine.

Not sure we'll necessarily need 3; really depends on the submissions we get & the quality thereof.

**Comment by s520** [comment]
OK.

Another potential issue is the copyright and license of the draft logo itself.
We need to transfer the copyright of the draft logo to the project or decide on a license.

**Comment by BosoViewExpress** [comment]
Will the BVE5 route paraser release?

**Comment by leezer3** [comment]
Not at the minute.
It's probably about 50% complete.1

**Comment by BosoViewExpress** [comment]
May I try this?
